modules/ROOT/pages/runtime-configure-gov.adoc

```diff
@@ -0,0 +1,83 @@
+= To Configure the Mule Runtime to Connect to the Government Cloud
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+To use Mule Runtime within the Government Cloud, you must redefine the configuration properties that enable Mule Runtime to point to the EU control plane. You must also provide the Client ID and Client Secret of your environment.
+
+By default, the Mule Runtime is configured to point to the US control plane. If you do not override these properties, the Mule Runtime modules are disabled in the EU control plane.
+
+The properties you must override depend on the version of the Mule runtime you are using, as described in the following sections.
+
+== Configuring Mule Runtime 4.0
+
+=== wrapper.conf
+
+To configure Mule Runtime using the `wrapper.conf` file, modify the following properties:
+
+----
+wrapper.java.additional.<n>=-Danypoint.platform.client_id=<environment client ID>
+wrapper.java.additional.<n>=-Danypoint.platform.client_secret=<environment client secret>
+wrapper.java.additional.<n>=-Danypoint.platform.base_uri=https://gov.anypoint.mulesoft.com
+wrapper.java.additional.<n>=-Danypoint.platform.analytics_base_uri=https://analytics-ingest.gov.anypoint.mulesoft.com
+----
+
+In the previous example, replace `<n>` with an integer value that is unique within the `wrapper.conf` file.
+
+=== Command Line
+
+To configure Mule Runtime from the command line, use the following options when starting the Mule runtime:
```

Comment:
Take into account that this applies to Unix-like systems. For Windows-based systems, "./" does not apply.

Reply:
Added some extra text - thank you!